typhos/utils.py

```diff
@@ -248,10 +247,10 @@ def use_stylesheet(dark=False, widget=None):
         # Load the path to the file
         style_path = os.path.join(ui_dir, 'style.qss')
         if not os.path.exists(style_path):
-            raise EnvironmentError("Unable to find Typhos stylesheet in {}"
```

Comment:
First I've noticed it:
```
The following exceptions are kept for compatibility with previous versions; starting from Python 3.3, they are aliases of [OSError](https://docs.python.org/3/library/exceptions.html#OSError).

exception EnvironmentError
```
https://docs.python.org/3/library/exceptions.html#EnvironmentError